app/models/user.rb

```diff
@@ -20,16 +20,11 @@ def authenticate(params)
     end
 
     def create(params)
-      begin
-        api_response = RestClient::Request.execute(method: :get, 
-          url: "https://www.pivotaltracker.com/services/v5/me", 
-          user: params[:username], 
-          password: params[:password])
-        token = JSON.parse(api_response)["api_token"]
-      rescue Exception => err
-        puts err.response
-        return
-      end
+      api_response = RestClient::Request.execute(method: :get,
+        url: "https://www.pivotaltracker.com/services/v5/me",
+        user: params[:username],
+        password: params[:password])
+      token = JSON.parse(api_response)["api_token"]
```

Comment:
This still has the same problem as before - I get an exception instead of a graceful failure without the begin/rescue. I'm changing it back for the time being. We can revisit this later if we find a graceful way to fail without the begin/rescue